packages/contracts/deployments/engine/V3/studio/dev/DEPLOYMENTS.md

```diff
@@ -0,0 +1,91 @@
+
+**Core Registry:** https://sepolia.etherscan.io/address/0x985C11541ff1fe763822Dc8f71B581C688B979EE#code
+
+Ownership on Core Registry transferred to the Engine Factory
+
+---
+
+# Batch Engine and Engine Flex Contract Deployments
+
+Date: 2024-05-09T17:01:48.023Z
+
+## **Network:** sepolia
+
+## **Environment:** dev
+
+**Engine Implementation:** https://sepolia.etherscan.io/address/0x00000000BB846ED9fb50fF001C6cD03012fC4485#code
+
+**Engine Flex Implementation:** https://sepolia.etherscan.io/address/0x00000000B33F6D5cA8222c87EAc99D206A99E17E#code
+
+**Engine Factory:** https://sepolia.etherscan.io/address/0x0000B005007298838aCF6589d4342920A9cB002a#code
+
+**Core Registry:** https://sepolia.etherscan.io/address/[object Promise]#code
+
+---
+
+**AdminACL:** https://sepolia.etherscan.io/address/0x37935f04d040F564337AE9BD76f2668664Fc9604#code
+
+        **Engine Contract Type:** Engine
+
+        **Engine Contract:** https://sepolia.etherscan.io/address/0xce9Fe7257F4C9AfB879564cC0bE03bFc7CAe85De#code
+
+        **Metadata**
+        - **Starting Project Id:** 0
+        - **Token Name:** dev-test2
+        - **Token Ticker:** DEV_TEST2
+        - **Auto Approve Artist Split Proposals:** true
+        - **Render Provider Address, Primary Sales:** 0x3c6412FEE019f5c50d6F03Aa6F5045d99d9748c4
+        - **Platform Provider Address, Primary Sales:** 0x0000000000000000000000000000000000000000
+        - **Null Platform Provider:** true
+        - **Allow Artist Project Activation:** true
+        - **Admin ACL Contract:** 0x37935f04d040F564337AE9BD76f2668664Fc9604
+        - **Super Admin Address:** 0x3c6412FEE019f5c50d6F03Aa6F5045d99d9748c4
+
+      **Other**
+
+      - **Starting project ID:** 0
+      - **Add initial project?:** false
+      - **Add initial token?:** false
+      - **Image Bucket:** dev-test2-sepolia
+
+        ---
+
+        **AdminACL:** https://sepolia.etherscan.io/address/0xc64949b06d59AF22b7182DD5B7DD0C8798E83a1D#code
+
+        **Engine Contract Type:** Engine Flex
+
+        **Engine Contract:** https://sepolia.etherscan.io/address/0xCeEE6DfA81687eE1c765B2396E337EB33F75c94D#code
+
+        **Metadata**
+        - **Starting Project Id:** 0
+        - **Token Name:** dev-test-flex
+        - **Token Ticker:** DEV_TEST_FLEX
+        - **Auto Approve Artist Split Proposals:** true
+        - **Render Provider Address, Primary Sales:** 0x3c6412FEE019f5c50d6F03Aa6F5045d99d9748c4
+        - **Platform Provider Address, Primary Sales:** 0x0000000000000000000000000000000000000000
+        - **Null Platform Provider:** true
+        - **Allow Artist Project Activation:** true
+        - **Admin ACL Contract:** 0xc64949b06d59AF22b7182DD5B7DD0C8798E83a1D
+        - **Super Admin Address:** 0x3c6412FEE019f5c50d6F03Aa6F5045d99d9748c4
+
+      **Other**
+
+      - **Starting project ID:** 0
+      - **Add initial project?:** false
+      - **Add initial token?:** false
```

Comment:
totally optional, but we could remove these as we will never add this capability imo

Reply:
great call ✅